### PR TITLE
Adding contact button to footer and removing from navbar

### LINKED ISSUE #330: Making a Contact US section at Footer [OTHER]
### What would you like to share?

image
This looks a little weird, should give them a better look

By Taking inspiration from this
image
Replacing location details with social icons

### PR DESCRIPTION
**Pull request by MANUSRAO**
# Issue  📐

- **I @MANUSRAO  have worked for #330**

[put x to check the boxes]: <> (This is a comment, it will not be included)

## Guidelines 🔐

**I accept the fact that i have followed the guidelines and have not copied the codes from around the internet**

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- **My pull request closes #330**

## Screenshots 📷

**Here are the pictures of changes that I have made 🔽**
Screenshot 2022-10-05 114529

  
  
  
---

### CODE REVIEW

**Review by IAmTamal — COMMENTED** [review]
<h1>It's great having you contribute to Milan !</h1> This is an <b>auto-generated</b> text<p>The <b>maintainers/owner</b> will look into the pull request soon, add relevant tags and proceed further, please have paitence. Meanwhile feel free to support <b>star the repo</b> and <b>share it</b> with your friends. :nerd_face: :rocket:</p>

**Review by IAmTamal — CHANGES_REQUESTED** [review]
The changes you made looks fantastic ! 

- Can we make it so that `contact us` is visible on the navbar on places where footer is not visible ? 
- Like the reg/sign in pages !

**Comment by MANUSRAO** [comment]
I think Vercel bot has not re deployed the site. Should I make another PR?

**Comment by IAmTamal** [comment]
> I think Vercel bot has not re deployed the site. Should I make another PR?

Donot worry about it

**Comment by IAmTamal** [comment]
> I think Vercel bot has not re deployed the site. Should I make another PR?

Yes, please do so. 
IDK something has glitched out maybe

**Comment by MANUSRAO** [comment]
Okay will do one PR soon

**Comment by IAmTamal** [comment]
> Okay will do one PR soon

Oh wait - it's working, lol

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -69,12 +69,19 @@ const Navbar = () => {
...
+                <li className="nav-item home">
+                  <Link to={"/contact"}>Contact</Link>
+                  <div
+                    className={
+                      "" +
+                      (location.pathname === "/contact" ? "active-link" : "")
+                    }
+                  ></div>
+                </li>
+              )}
```
Comment (IAmTamal) [inline src/components/Navbar.jsx:85]: You need to make it visible for `/clubs/register` and `/clubs/login` too

Comment (MANUSRAO) [reply src/components/Navbar.jsx:85]: Will do the changes soon!

**Review by IAmTamal — APPROVED** [review]
## Looks good to me ✅

- **⚒ Thankyou for working on the issue**
- **⭐ Do star the repo, to get updates on more issues !**
- **💚 Keep contributing to Opensource**

**Pull request merged**